lib/src/widgets/utils/ds_user_avatar.widget.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSUserAvatar extends StatelessWidget {
+  final String text;
+  final double radius;
+  final Color backgroundColor;
+  final Color textColor;
+  const DSUserAvatar({
```

Comment:
Add a line break between constructor and fields/properties. The Dart extension's auto format should do the work.